tools/tileGet.py

```diff
@@ -82,25 +89,48 @@ def retrieveMapServerTiles(options, west, south, east, north, decals, net):
         sx, sy = fromLatLonToTile(north, west, zoom)
         ex, ey = fromLatLonToTile(south, east, zoom)
         numTiles = (ex - sx + 1) * (ey - sy + 1)
+        
+    # opener = urllib.build_opener()
+    # opener.addheaders = [('User-agent', 'Mozilla/5.0')]
+    # urllib.install_opener(opener)
+    
+    # N = (ex - sx + 1) * (ey - sy + 1)
+    
+    if options.parallel_jobs != 0:
+        original_sigint_handler = signal.signal(signal.SIGINT, signal.SIG_IGN)
+        pool = Pool(options.parallel_jobs)
+        signal.signal(signal.SIGINT, original_sigint_handler)
+    
```

Comment:
What is the signal handling for?

Reply:
Just out of curiosity. This does not happen with the previous implementation with concurrent.futures?

Reply:
No it did not happen. `concurrent.futures` works just fine without the hack